api-report/azure-client.api.md

```diff
@@ -73,6 +77,18 @@ export interface AzureMember<T = any> extends IMember {
     userName: string;
 }
 
+// @public (undocumented)
+export class AzureUrlResolver implements IUrlResolver {
+    constructor();
+    // (undocumented)
+    getAbsoluteUrl(resolvedUrl: IResolvedUrl, relativeUrl: string): Promise<string>;
+    // (undocumented)
+    resolve(request: IRequest): Promise<IFluidResolvedUrl>;
+}
+
+// @public (undocumented)
+export const createAzureCreateNewRequest: (ordererUrl: string, storageUrl: string, tenantId: string) => IRequest;
+
```

Comment:
I don't want this exposed to the end developer unless we have a compelling scenario. Does this need to be exported from the package?

Reply:
Updated `index.ts` so that the helper function isn't exported